### DISCUSSION TITLE
Mason available does not show everything

### DISCUSSION THREAD

**Discussion by Kim-Robin** [mason-org/mason.nvim#1054] [Q&A] [answered]
I was trying to install some lsp on nvim-lspconfig using mason.nvim

however, I was not able to find one in Available even though it is listed on 
https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md

How can I install lsp that is not listed on mason but listed on nvim-lspconfig?

**Comment by mrtnvgr** [comment, marked as answer]
Mason can't install what it does not support, use other installation ways (i.e. use another package manager, or install manually).

**Reply by Kim-Robin** [reply]
Is there any plan to add more lsp support? I like mason a lot so I hope this can be support every language

**Reply by mrtnvgr** [reply]
> Is there any plan to add more lsp support? I like mason a lot so I hope this can be support every language

It's impossible to have support for every language. If something isn't supported yet, you can make a PR/issue. 🤗
